addons/event/data/event_demo.xml

```diff
@@ -5,27 +5,6 @@
              <field name="groups_id" eval="[(4, ref('event.group_event_user'))]"/>
      </record>
 
-    <!-- Demo Data for Event Type -->
```

Comment:
I will revert the whole change in event. It makes no sense to have some kind of default product for completely different event and tickets. Products are linked to tickets, not categories.